klaytn-tx-latency-measurement/.env.template

```diff
@@ -1,5 +1,6 @@
 PRIVATE_KEY=
 CAVER_URL=https://public-node-api.klaytnapi.com/v1/baobab
+UPLOAD_METHOD=AWS
```

Comment:
Are GCP_PROJECT_ID and GCP_KEY_FILE_PATH going to be added here?

Reply:
Gee, thanks